tools/coqdoc/cpretty.mll

```diff
@@ -1178,6 +1184,9 @@ and skip_to_dot_or_brace = parse
   | _ { skip_to_dot lexbuf }
 
 and body_bol = parse
+  | space_nl+ ('-'+ | '+'+ | '*'+) space*
+      { String.iter Output.char (lexeme lexbuf);
+        if is_none !formatted then true else body_bol lexbuf }
```

Comment:
Is that really the best place to handle this? (And it seems to be missing the `Output.indentation` and newline increment.)

Reply:
OK I see the idea though maybe the regex should be `space* ('-`+ | ...) space*` to let newlines be handled elsewhere. Is there also not some bad interaction with negative numbers?